src/person.rb

```diff
@@ -1,13 +1,19 @@
+require_relative './nameable'
+require_relative './capitalize'
+require_relative './trimmer'
+require_relative './rental'
+
 class Person < Nameable
-  attr_accessor :name, :age
-  attr_reader :id
+  attr_accessor :name, :age, :rental
+  attr_reader :id, :parent_permission
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rental = nil
```

Comment:
- [ ] Kindly notice that it is better to rename it to `rentals` instead of `rental` as you are going to add many rentals to it.
- [ ] Also notice that it is better to make the `rentals` equal to empty array `[]` so it makes it easy to push data in it.

```ruby
@rentals = []
```